.github/workflows/script-evaluation-test.yml

```diff
@@ -4,6 +4,10 @@ on:
   schedule:
     - cron: 30 3 * * * # 3:30am every day 
 
+concurrency:
+  group: script-evaluation-test
+  cancel-in-progress: true
```

Comment:
This seems okay... we shouldn't ever end up with multiple of these in flight? Worth a comment.